media-sound/pasystray/pasystray-0.6.0.ebuild

```diff
@@ -11,35 +11,46 @@ SRC_URI="https://github.com/christophgysin/${PN}/archive/${P}.tar.gz"
 LICENSE="LGPL-2.1"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="libnotify"
+IUSE="libnotify plasma"
 
 RDEPEND="
 	>=media-sound/pulseaudio-5.0-r3[glib,zeroconf]
 	>=net-dns/avahi-0.6
+	>=dev-libs/glib-2.48.2
 	x11-libs/gtk+:3
 	x11-libs/libX11
 	libnotify? ( >=x11-libs/libnotify-0.7 )
+	plasma? ( dev-libs/libappindicator:3 )
```

Comment:
What is the relation of USE=plasma to libappindicator? This is the wrong place to fix a package's system tray icon within Plasma - it is done by switching on gtk3 in plasma-desktop itself.